dashboard/app/handler.go

```diff
@@ -61,7 +61,8 @@ func handleContext(fn contextHandler) http.Handler {
 			logf(c, "%v", err)
 			w.WriteHeader(http.StatusInternalServerError)
 			if err1 := templates.ExecuteTemplate(w, "error.html", data); err1 != nil {
-				http.Error(w, err.Error(), http.StatusInternalServerError)
+				combinedError := fmt.Errorf("got err \"%v\" processing ExecuteTemplate() for err \"%v\"", err1, err)
+				http.Error(w, combinedError.Error(), http.StatusInternalServerError)
```

Comment:
Would there be any noticeable difference if it was just

```
errorString := fmt.Sprintf("got err \"%v\" processing ExecuteTemplate() for err \"%v\"", err1, err)
http.Error(w, errorString, http.StatusInternalServerError)
```

?

Reply:
Your one is lighter. Don't see any side effects.